### PR TITLE
Adding a new navigation for outstanding balance members.

### PR DESCRIPTION
**Pull request by darkodosenovic**
google chrome - stackoverflow com - - screen shot june 20 2017 at 9 27 am
🐘 🎨

Creating a new navigation for outstanding balance members

### CODE REVIEW

File: spec/ama_layout/navigation_spec.rb [hunk 1]
```diff
@@ -156,5 +156,24 @@
         end
       end
     end
+
+    context "member-with-outstanding-balance" do
+      context "subnavs" do
+        context "driver education" do
```
Comment (rubene) [inline spec/ama_layout/navigation_spec.rb]: can you add a context for the renew links?

**Comment by rubene** [comment]
increase version

**Comment by rubene** [comment]
💩

**Comment by darkodosenovic** [comment]
:wave:

**Comment by darkodosenovic** [comment]
:wave:

**Comment by rubene** [comment]
🐘 🐘 🐘

**Comment by sineade** [comment]
:art: :art: :art:

I know the banner isn't in this PR but I just noticed the last sentence where it says "pay now", is that a link? If so, maybe we should just add an underline for this because it's not obvious, and if it's not a link we might want to rethink our wording because when I see something like "pay now" I assume it will be a link because it sounds actionable.

**Comment by darkodosenovic** [comment]
@sineade  Agree, changing it now

**Pull request merged**